Makefile

```diff
@@ -191,6 +191,8 @@ install: go.install
 
 check test: go.test
```

Comment:
Sounds good. However the dependency wont be here. It will be added as a dependency in the go.test. https://github.com/rook/rook/blob/master/build/makelib/golang.mk#L154